org.rncbc.qsynth.json

```diff
@@ -0,0 +1,50 @@
+{
+  "app-id": "org.rncbc.qsynth",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.2",
+  "sdk": "org.kde.Sdk",
+  "command": "qsynth",
+  "rename-icon": "qsynth",
+  "rename-desktop-file": "qsynth.desktop",
+  "rename-appdata-file": "qsynth.appdata.xml",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc",
+    "--socket=x11",
```

Comment:
does this not work properly under wayland? It's built against Qt 6 so it should be really solid there. I'd recommend enabling wayland and switching x11 to fallback-x11

Reply:
Done, thanks!